src/graph/serialize/heterograph_serialize.cc

```diff
@@ -1,7 +1,7 @@
 /**
  *  Copyright (c) 2019 by Contributors
  * @file graph/serialize/heterograph_serialize.cc
- * @brief DGLHeteroGraph serialization implementation
```

Comment:
Let's not modify C/C++ files for the time being.

Reply:
Make sense. Let me revert it.